corehq/toggles.py

```diff
@@ -1944,6 +1944,19 @@ def _commtrackify(domain_name, toggle_is_enabled):
     help_link="https://confluence.dimagi.com/display/saas/Limited+Login+As",
 )
 
+DISABLE_EMAIL_REPORTS_PERMISSION = StaticToggle(
+    'disable_email_reports',
+    'USH: Disable emailing and downloading of reports',
```

Comment:
This sounds like it disables a feature for the whole domain, maybe include "permission" in the name?